app/src/db/MissedPingsMq.js

```diff
@@ -40,6 +42,23 @@ const MissedPingsMq = {
     console.log(job);
   },
 
+  async populateStartSoloQueues() {
+    const monitors = await dbGetAllMonitors();
+
+    for (const monitor of monitors) {
+        switch (monitor.type) {
+            case 'solo':
+                await MissedPingsMq.addSoloJob({monitorId: monitor.id}, calculateDelay(monitor));
+                break;
+            default:
+                await MissedPingsMq.addStartJob({monitorId: monitor.id}, calculateDelay(monitor));
+                break;   
+        }
+    }
```

Comment:
Could we use `Promise.all` here?

Reply:
That was a good idea 💡 I ended up using `Promise.allSettled` because the jobs aren't dependent on each other –  my understanding is that if a job errored out and we had used `Promise.all`, it could cause the rejection of the other jobs if they didn't finish faster than the errored out job.

Just to show that I tested (although it does look exactly the same as the earlier screenshot, ha)
![Screenshot 2023-10-26 at 8 07 42 PM](https://github.com/Sundial-Inc/server/assets/10123446/d8736acd-6f97-46c1-a57c-001d76c5b712)

Reply:
Nice, good idea using `Promise.allSettled`!